File: r/raindrops/raindrops.R
raindrops <- function(n) {
  res = ""
  if (n %% 3 == 0) {
    res = paste(res,"Pling",sep = "")
  }
  if (n %% 5 == 0) {
    res = paste(res,"Plang",sep = "")
  }
  if (n %% 7 == 0) {
    res = paste(res,"Plong",sep = "")
  }
  if (res == "") {
    return (toString(n))
  } else {
    return (res)
  }
}

File: solutions/python/raindrops/1/raindrops.py
def convert(n = None):
    res = ""
    if n % 3 == 0:
        res += "Pling"
    
    if n % 5 == 0:
        res += "Plang"
    
    if n % 7 == 0:
        res += "Plong"
    
    if res == "":
        return str(n)
    else:
        return res

